tests/trafficmanagement/scripts/request_timeouts.txt

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source ${REPO_ROOT}/content/en/docs/tasks/traffic-management/request-timeouts/snips.sh
+
+# Export HOST, PORT and URL
+export INGRESS_HOST={{ .ingressHostCom }}
+export INGRESS_PORT={{ .ingressPortCom }}
+export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+
+# Step 0: route all to v1
+# $snippet
+snip_before_you_begin_1
+# $verify
+virtualservice.networking.istio.io/productpage created
+virtualservice.networking.istio.io/reviews created
+virtualservice.networking.istio.io/ratings created
+virtualservice.networking.istio.io/details created
+# $endsnippet
+
+# Step 1: Route requests to v2 of the reviews service
+snip_request_timeouts_1
+
+istioctl experimental wait --for=distribution VirtualService reviews.default
+
+# Step 2: Add a 2 second delay to calls to the ratings service
+snip_request_timeouts_2
+
+istioctl experimental wait --for=distribution VirtualService ratings.default
+
+echo "sending request to $GATEWAY_URL"
+
+# Step 3: Verify 2s delay with ratings stars displayed
+# $snippet
+curl -sS http://"${GATEWAY_URL}"/productpage | grep -o ".*Reviewer1"
+sleep 2
```

Comment:
This doesn't make sense. The previous curl command will delay for 2 seconds before returning. So sleeping for another 2 seconds after that isn't necessary or useful.
```suggestion
```